README.md

```diff
@@ -10,3 +10,39 @@ Also add a picture of the structure to this [README](README.md) file.
 
 ### Well Entrance
 ![Well Entrance](https://github.com/Terasology/Mineshafts/raw/master/preview/WellEntrance.jpg)
+
+# This is for my GCI Task (I) - Structure Templates.
```

Comment:
can you remove all this extra information about your gci task. format your images similarly to how it's formatted in the main repository. If you really want the description you can add it below the image.

Reply:
Sir see below , I have done changes and updated it,